.github/workflows/uats.yaml

```diff
@@ -13,20 +13,35 @@ jobs:
         uat:
           - uats-namespace-isolation
           - uats-ingress
+          - uats-cos
       fail-fast: false
     concurrency:
       group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.uat }}
       cancel-in-progress: true
 
     steps:
+      - name: Maximize disk pace
```

Comment:
You could use this:

```
      - name: Maximise GH runner space
        uses: jlumbroso/free-disk-space@v1.3.1
```

or self-hosted runners, I'm fine either way.

Reply:
using the action instead in https://github.com/canonical/charmed-temporal-uats/pull/7/commits/63276c53bc676850aeead86288ca3dd54efc70dc

will switch to self hosted if it helps stabilize the tests further